config/ec2/config.go

```diff
@@ -327,4 +327,40 @@ func Configure(p *config.Provider) {
 	p.AddResourceConfigurator("aws_ec2_traffic_mirror_target", func(r *config.Resource) {
 		delete(r.References, "network_load_balancer_arn")
 	})
+
+	p.AddResourceConfigurator("aws_vpc_ipam_pool", func(r *config.Resource) {
+		r.References["ipam_scope_id"] = config.Reference{
+			Type: "VPCIpamScope",
+		}
+	})
+
+	p.AddResourceConfigurator("aws_vpc_ipam_scope", func(r *config.Resource) {
+		r.References["ipam_id"] = config.Reference{
+			Type: "VPCIpam",
+		}
+	})
+
+	p.AddResourceConfigurator("aws_ami", func(r *config.Resource) {
+		r.References["ebs_block_device.snapshot_id"] = config.Reference{
+			Type: "EBSSnapshot",
+		}
+	})
+
+	p.AddResourceConfigurator("aws_ami_copy", func(r *config.Resource) {
+		r.References["source_ami_id"] = config.Reference{
+			Type: "github.com/upbound/provider-aws/apis/ec2/v1beta1.AMI",
```

Comment:
Because they are in the same package, it should be sufficient to only use the type name without qualifying it with the package:
```suggestion
			Type: "AMI",
```

If you make configuration/example manifest changes, please don't forget to rerun uptest on them to validate the latest state and update the PR description. But with the change suggested here, I don't expect any changes in the example manifests or the resolver's generated as we are effectively referring to the same type. So, if I'm not mistaken, we should be good without retesting. Could you please also verify?

Reply:
Done